mono/utils/mono-mmap.c

```diff
@@ -51,9 +51,11 @@
 #define MAP_ANONYMOUS MAP_ANON
 #endif
 
+#if !defined(__APPLE__)  // returning virtual addresses <4G requires entitlement on Apple platforms, do not use it
 #ifndef MAP_32BIT
 #define MAP_32BIT 0
 #endif
+#endif
```

Comment:
I don't think this is needed here.

Reply:
Yeah it's mostly just additional documentation and foolproofing for someone copying this code.